juntagrico/entity/jobs.py

```diff
@@ -233,6 +237,14 @@ class Meta:
 class RecuringJob(Job):
     type = models.ForeignKey(JobType, on_delete=models.PROTECT, verbose_name=_('Jobart'))
     additional_description = models.TextField(_('Zusätzliche Beschreibung'), max_length=1000, blank=True, default='')
+    duration_override = models.PositiveIntegerField(
+        _('Besondere Dauer in Stunden'), null=True, blank=True, default=None,
```

Comment:
i would not call it besondere, somethin more hinti that is is an override

Reply:
tried my best